test/server/README.md

```diff
@@ -1,13 +1,12 @@
 This test suite validates PHP calls and MySQL database content.
 
-*** Instructions
+\*\*\* Instructions
```

Comment:
This looks weird - is this definitely right?

Reply:
The very opiniated prettier thinks so ;)

Reply:
I've never seen it do this. Let me take a quick look.

Reply:
OK, the original is wrong - I thought it was hashes.

```suggestion
### Instructions
```